packages/keyValueCache/package.json

```diff
@@ -19,6 +19,6 @@
   "license": "MIT",
   "dependencies": {
     "@apollo/utils.logger": "^1.0.0",
-    "lru-cache": "^7.10.1"
+    "lru-cache": "^7.10.1 - 7.13.1"
```

Comment:
@trevor-scheer yarn berry doesn't like this notation. According to https://semver.npmjs.com/ the syntax is invalid.

<img width="474" alt="image" src="https://user-images.githubusercontent.com/1404810/203509423-6675a523-a058-40dc-b5ee-dc20377887c4.png">

Reply:
Thanks @SimenB I'll look into this!